api/src/Controller/BaseController.php

```diff
@@ -346,6 +366,23 @@ public function findMemberId(
 
     }
 
+    
+    public function notFoundResponse(
```

Comment:
I'm leaving this for now, even though I'm not using it as much as I thought I would. Really, what I want to do is add an exception handling in `__invoke` so we can just throw `NotFound` exceptions and get rid of a lot of repeated code. But that's DEFINITELY another PR.

Reply:
I like that idea. Let move toward that.